src/main/scala/chisel3/tester/experimental/TestOptionBuilder.scala

```diff
@@ -15,7 +17,14 @@ package object TestOptionBuilder {
 
     @deprecated("Use withAnnotations instead")
     def withExecOptions(manager: ExecutionOptionsManager with HasTreadleSuite): ChiselScalatestTester#TestBuilder[T] = {
-      new x.outer.TestBuilder[T](x.dutGen, manager.toAnnotationSeq)
+      val annos = manager.toAnnotationSeq.map {
+        case CompilerAnnotation(compiler) if compiler.isInstanceOf[LowFirrtlCompiler] => TreadleBackendAnnotation
+        case CompilerAnnotation(compiler) if compiler.isInstanceOf[NoneCompiler]      => TreadleBackendAnnotation
+        case CompilerAnnotation(_)                                                    => VerilatorBackendAnnotation
```

Comment:
Why is this the default? Wouldn't it make more sense to error out on an unknown backend instead of having this perhaps surprising behavior?

Reply:
Made the list specific for all verilog and treadle.